### ISSUE TITLE
[features] Sorting books

### ISSUE THREAD

**Issue by kaazimh** [PaulWoitaschek/Voice#810]
Please include the following details in your bug report:

App-Version: 4.02
Phone Name: Lenovo P2
Android Version: 7.0

I do not seem to have any option for sorting. It is not showing on the options screen.

Could this be put in, and is it possible to sort books by folder path? It would be easy to organise books by author and series then.
Thanks

**Comment by droidberg** [comment]
yes, i made the mistake of doing a clean install and all my hundreds of books are just scattered! lol, it was a bit nerve wrecking!

**Comment by droidberg** [comment]
i cant believe this has had no attention. does no one else have the issue?

**Comment by droidberg** [comment]
@PaulWoitaschek 

sorry to complain but has anyone else mentioned books not being in alphabetical order?

**Comment by PaulWoitaschek** [comment]
@droidberg 
I understand that this issue is important to you, but one comment is enough.

The default sorting is not random but:
* For current books sorted by last played
* For new books by date added
* For completed books by last played

Most of the users don't have hundreds of books so they are not as affected by this as you.
